homeassistant/components/tuya/climate.py

```diff
@@ -187,9 +256,53 @@ def supported_features(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return self._tuya.min_temp()
+        min_temp = (
+            self._min_temp if self._min_temp is not None else self._tuya.min_temp()
+        )
+        if min_temp is not None:
+            return min_temp
+        return super().min_temp
 
     @property
     def max_temp(self):
         """Return the maximum temperature."""
-        return self._tuya.max_temp()
+        max_temp = (
+            self._max_temp if self._max_temp is not None else self._tuya.max_temp()
+        )
+        if max_temp is not None:
+            return max_temp
+        return super().max_temp
+
+    def _get_ext_temperature(self):
+        """Get external temperature entity current state."""
+        if not self._temp_entity or self._temp_entity == ENTITY_MATCH_NONE:
+            return None
+
+        def _log_error(error_msg):
+            if not self._temp_entity_error:
+                _LOGGER.warning(
+                    "Error on Tuya external temperature sensor %s: %s",
+                    self._temp_entity,
+                    error_msg,
+                )
+                self._temp_entity_error = True
```

Comment:
It would be a bit faster to make this `self._log_temp_error` since every time this is called it has to recreate the function.

Reply:
Done